Calculator/Program.cs

```diff
@@ -0,0 +1,191 @@
+﻿using CalculatorLibrary;
+using System.Text.RegularExpressions;
+
+namespace CalculatorProgram
```

Comment:
Try this to remove the huge amount of unused space on the left side of the file: 

![image](https://user-images.githubusercontent.com/108711476/203434712-05522969-b1a6-4c84-829a-82a063da3481.png)